Services/TokenValidationService.cs

```diff
@@ -100,6 +100,10 @@ public TokenValidationService(IOptions<AppSettings> settings, ILogger<TokenValid
                 // Get tenant ID and client ID
                 var tenantId = _settings.TenantId;
                 var clientId = _settings.ClientId;
+                var authUri= string.IsNullOrEmpty(_settings.AuthUrl) 
```

Comment:
authorityUri will be a required setting going forward and you wouldn't need to do an if/else check right?